src/components/Posts/PostDeleteConfirmationModal.tsx

```diff
@@ -70,8 +71,20 @@ const PostDeleteModal = ({
       } else {
         dispatch(deletePost(postId));
       }
+      dispatch(
+        enqueueToast({
+          message: "Your post has been deleted",
+          severity: "info",
+        })
+      );
     } catch (error) {
       console.error("Failed to delete the post", error);
+      dispatch(
+        enqueueToast({
```

Comment:
I don't have a strong opinion either way, so I can remove it if we want. I can increase the `autoHideDuration` or change the message if we want, do you have any suggestions?

I have seen places where toasts are used for errors too though, for example when the connection is poor. I think any user feedback is better than nothing, but we're kinda ignoring errors as it is anyways. So again I'm down for anything

Reply:
You're right in saying any feedback is better than nothing. Let's stick to it, I suggest increasing the `autoHideDuration` for error messages. Maybe `5000` works here like you had originally?

Reply:
Sounds good, I made the default `autoHideDuration` 5000 if severity is error in `toastSlice.ts` so that we don't have to remember to manually set it everywhere